blocks/handler.go

```diff
@@ -71,7 +71,8 @@ func (h *Handler) HandleBlockData(ctx context.Context, data []byte) error {
 		logger.Info("known block")
 		return nil
 	}
-	logger.With().Info("new block", log.Inline(&b))
+	logger.With().Info("new block ID", log.String("block_id", b.ID().AsHash32().ShortString()))
```

Comment:
no need to do log.String on the IDs. just 

`logger.With().Info("new block", b.ID())`. all types implement `Field()` for logging